### ISSUE TITLE
Arabic and Farsi language not set

### ISSUE THREAD

**Issue by mojtabas74** [vector-im/element-android#2529]
when change language to Farsi or Arabic app crash and language not changed

**Comment by bmarty** [comment]
I notice there are lots of issue in the translation regarding the templates in Farsi and Arabic. On my side it is not crashing, but it will crash if a string with a problem is used.
Can you provide some crash log?

**Comment by mojtabas74** [comment]
It's not crash my mistake, but  when i change language to Farsi noting change and when i press back in setting app closed.

**Comment by bmarty** [comment]
Ok. Yes, we know that changing the language restarts the application. This is a side effect of a security issue we tried to fix.
But if launching again the app, the language should have been changed.

**Comment by mojtabas74** [comment]
yes, app restart, but language not change to Farsi or Arabic other language is ok and changed but Farsi and Arabic just change date .
